src/defaults/workflowSpecification.tsx

```diff
@@ -0,0 +1,277 @@
+import {
+  CircleCheck,
+  CircleDot,
+  CircleX,
+  BadgeCheck,
+  type LucideIcon
+} from '@ttab/elephant-ui/icons'
+
+export interface WorkflowItem {
+  title: string
+  description: string
+}
+
+export interface WorkflowTransition extends WorkflowItem {
+  verify?: boolean
+  default?: boolean
+}
+export interface WorkflowState extends WorkflowItem {
+  transitions: Record<string, WorkflowTransition>
+}
+
+export type WorkflowSpecification = Record<string, WorkflowState>
+
+export interface StatusSpecification {
+  icon: LucideIcon
+  className: string
+}
+
+export const StatusSpecifications = {
+  draft: {
```

Comment:
Maybe
```suggestion
export const StatusSpecifications: Record<string, StatusSpecification> = {
```
instead of typecasting it?

Reply:
Better stick to the regular pattern. Agree!